analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -701,7 +726,35 @@ def main(args):
         metadata['result_source_files'] = \
             metadata_prev['result_source_files']
 
-    analyzer.perform_analysis(args, skip_handler, context, actions, metadata)
+    CompileCmdParseCount = \
+        collections.namedtuple('cmpCmdParseCount',
```

Comment:
As I see in docs (https://docs.python.org/2/library/collections.html#collections.namedtuple), the `typename` parameter is camel cased and starting with an upper case letter, and also it is the same as the variable in which we store the return value.

```suggestion
        collections.namedtuple('CompileCmdParseCount',
```